install.sh

```diff
@@ -100,16 +101,21 @@ function curl () {
 }
 
 case "${ARCH}" in
-"i686"|"x86_64"|"armv7l"|"aarch64")
+"i686")
+  echo_error "Installs on i686 devices are no longer supported by Chromebrew. :/"
+  exit 1
+  ;;
+"x86_64"|"armv7l"|"aarch64")
   LIB_SUFFIX=
   # See https://superuser.com/a/1369875
   # If /bin/bash is 64-bit, then set LIB_SUFFIX, as this is true on both
   # x86_64 and aarch64 userspace
   # shellcheck disable=SC2046
   [ $(od -An -t x1 -j 4 -N 1  /bin/bash) == 02 ] && LIB_SUFFIX='64'
   if [[ $LIB_SUFFIX == '64' ]] && [[ $ARCH == 'aarch64' ]]; then
-    echo_error "Your device is not supported by Chromebrew yet :/"
-    exit 1
+    echo_error "Your device is not supported by Chromebrew yet, installing as armv7l."
+    LIB_SUFFIX=
+    ARMV7LONAARCH64=1
```

Comment:
Does this logic even work?  I've noticed several issues appear due to problems installing on aarch64 devices.

Reply:
It worked in the M113 container!

Reply:
Still works in the container...